db/blob/blob_file_reader_test.cc

```diff
@@ -181,8 +239,9 @@ TEST_F(BlobFileReaderTest, CreateReaderAndGetBlob) {
     uint64_t bytes_read = 0;
 
     ASSERT_TRUE(reader
-                    ->GetBlob(read_options, key, blob_offset - 1, blob_size,
-                              kNoCompression, &value, &bytes_read)
+                    ->GetBlob(read_options, keys[2], blob_offsets[2] - 1,
```

Comment:
Very minor but shouldn't this be the other way round? I.e. `blob_offsets[0] - 1` would be too close to the start and `blob_offsets[2] + 1` too close to the end?

Reply:
Done